Update file1.adoc (#15)
alxlenc/pygh-tests: content/file1.adoc

## This is a Title
I create a file.

> A link to the list of drivers and their snapshot support is provided in the references section.

Reviewer: How about...
```suggestion
I have provided the link to the list of drivers, which includes their snapshot support, in the references section.
```
